fix formatting of review txs value

ericvalcik opened the pull request:
## Description

Fix wrapping of Coin/Token symbol in the `TransactionReviewSummary.tsx`.

## Related Issue

Resolve https://github.com/trezor/trezor-suite/issues/13603

## Screenshots:

before:
<img width="864" alt="image" src="https://github.com/user-attachments/assets/1fda6ce6-0a5f-4578-a827-c2de45ce4be9">


after changes:
<img width="865" alt="image" src="https://github.com/user-attachments/assets/d3100cb2-8496-4b7a-ae21-ca78f127cda2">

enjojoy commented:
Please squash the commits and use conventional commits style for the commit message

tomasklim commented on packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewSummary.tsx:
```diff
@@ -69,7 +69,7 @@ const Headline = styled.div`
     font-weight: 600;
     margin-top: 20px;
-    word-break: break-all;
+    word-break: normal;
```
`break-word` would fix overflowing
Some tokens can have 18 decimals and amount can be also huge
<img width="323" alt="Screenshot 2024-09-08 at 8 17 51" src="https://github.com/user-attachments/assets/b010349e-5402-4f33-9eb0-94641c4cfa1e">

tomasklim commented on packages/suite/src/components/suite/FormattedCryptoAmount.tsx:
```diff
@@ -90,7 +90,7 @@ export const FormattedCryptoAmount = ({
             {!!signValue && <Sign value={signValue} />}
             <Value data-testid={dataTest}>{formattedValue}</Value>
-            {formattedSymbol && <BlurUrls text={'\u00A0' + formattedSymbol} />}
+            {formattedSymbol && <BlurUrls text={' ' + formattedSymbol} />}
```
nit:
```suggestion
            {formattedSymbol && <>
              {' '}
              <BlurUrls text={formattedSymbol} />
            </>}
```
Something like that would look better

tomasklim reviewed (changes requested):
Hello, thanks for contribution. I suggest you to create branches in your fork instead of commiting to develop

ericvalcik commented:
Thanks for the comments @tomasklim. I implemented the comments and also moved the changes to a different branch, which made me create a new PR, so please let's continue the review process here -> https://github.com/trezor/trezor-suite/pull/14216
